unpackaged/config/regression_bdi_custom_metadata/customMetadata/npsp__Data_Import_Object_Mapping.CustomObject1_7b96c6e1b_3cc49eec4.md

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomMetadata xmlns="http://soap.sforce.com/2006/04/metadata" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <label>CustomObject1</label>
+    <protected>false</protected>
+    <values>
+        <field>npsp__Custom_Mapping_Logic_Class__c</field>npsp__
+        <value xsi:nil="true"/>
+    </values>
+    <values>
+        <field>npsp__Data_Import_Object_Mapping_Set__c</field>npsp__
+        <value xsi:type="xsd:string">Migrated_Custom_Object_Mapping_Set</value>
+    </values>
+    <values>
+        <field>npsp__Imported_Record_Field_Name__c</field>npsp__
+        <value xsi:type="xsd:string">npsp__CustomObject1Imported__c</value>
```

Comment:
@davidmreed I was just testing this out and realized we have some additional issues with namespacing that I didn't pick up on previously. For any of the custom unpackaged schema related to these records - both sobjects and fields - we need to strip the namespace information. However we still need the namespace for any references to packaged schema. I can think of some somewhat laborious ways to accomplish this, but before I do, anything brilliant from your end?

Reply:
@ptewson-sfdo Did you extract it from a namespaced org? Or a managed one?

Reply:
Originally from a non-namespaced org, but then I manipulated namespaces programmatically - all part of an attempt to avoid having to manually set up all this mapping which would be laborious and error-prone. However, there may not be any way to avoid it given this new limitation

Reply:
I don't think there's a particularly good way to do that. It might be easier to re-extract the non-namespaced form and then selectively add the namespace than to try to selectively remove it, but I'm not sure either way.